snapcraft.yaml

```
@@ -0,0 +1,35 @@
+name: jd
+base: core24
+adopt-info: jd
+summary: Utility to diff and patch json and yaml files
+description: |
+  jd is a commandline utility and Go library for diffing and patching JSON
+  and YAML values. It supports a native jd format (similar to unified format)
+  as well as JSON Merge Patch (RFC 7386) and a subset of JSON Patch (RFC 6902).
+  Try it out at http://play.jd-tool.io/.
+
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
```

Again, moving the base here makes more sense to me. The comments are somewhat unnecessary in my opinion, and `lzo` compression works faster than the default `xz` compression (though it may be a very fractional change for a command-line utility).

```suggestion
base: core24
grade: stable
confinement: strict
compression: lzo
```